source/common/event/dispatcher_impl.cc

```diff
@@ -36,6 +38,12 @@
 
 namespace Envoy {
 namespace Event {
+namespace {
+// Our tracked object stack likely won't grow larger than this initial
```

Comment:
optional super nitty nit: minld preference for avoiding pronouns.  "The tracked object stack" "since the stack shouldn't have to grow larger"